webpack.local.conf.js

```diff
@@ -14,18 +15,18 @@ module.exports = {
 		}]
 	},
 	devServer: {
-		contentBase: './',
+		contentBase: './public',
 		port: 8080,
 		noInfo: false,
 		hot: true,
 		inline: true,
-		proxy: {
-			'/': {
-				bypass: function (req, res, proxyOptions) {
-					return '/public/index.html';
-				}
-			}
-		}
+		// proxy: {
```

Comment:
If it works, you can remove the code altogether. We have git history if we ever need to revert.